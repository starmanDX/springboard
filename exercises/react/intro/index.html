<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Part1</title>
</head>
<body>
    <div id="root"></div>
    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
    <script src="FirstComponent.jsx" type="text/jsx"></script>
    <script src="NamedComponent.jsx" type="text/jsx"></script>
    <script src="App.jsx" type="text/jsx"></script>
</body>
</html>