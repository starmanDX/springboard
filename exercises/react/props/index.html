<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>JSX Part2</title>
</head>
<body>
    <h1>JSX Props Part 2 - Tweets</h1>
    <div id="root"></div>
    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
    <script src="Tweet.jsx" type="text/jsx"></script>
    <script src="App.jsx" type="text/jsx"></script>
</body>
</html>