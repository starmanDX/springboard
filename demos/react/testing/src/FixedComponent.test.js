import React from "react";
import { render } from "@testing-library/react";
import FixedComponent from "./FixedComponent";

//SMOKETEST
test("it renders without crashing", () => {
  render(<FixedComponent />);
});

test("it matches snapshot", () => {
  const { asFragment } = render(<FixedComponent />);
  expect(asFragment()).toMatchSnapshot();
});
