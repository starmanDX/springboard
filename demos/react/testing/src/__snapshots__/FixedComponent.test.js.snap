// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <p>
    Hello, I'm a FixedComponent!
  </p>
  <p>
    Here are some numbers:
  </p>
  <p>
    42
  </p>
  <p>
    43
  </p>
  <p>
    44
  </p>
  <p>
    45
  </p>
</DocumentFragment>
`;
